src/node.cc

```diff
@@ -1030,7 +1030,7 @@ InitializationResult InitializeOncePerProcess(
   }
 
   if (init_flags & kInitOpenSSL) {
-#if HAVE_OPENSSL
+#if HAVE_OPENSSL && !defined(OPENSSL_IS_BORINGSSL)
```

Comment:
Please update line 1094 accordingly:

```
#endif  // HAVE_OPENSSL
```

(GitHub should really allow adding comments to other lines...)